src/nimble_kokkos.cc

```diff
@@ -372,19 +372,19 @@ int ExplicitTimeIntegrator(const nimble::Parser & parser,
   nimble_kokkos::DeviceVectorNodeView velocity_d = model_data.GetDeviceVectorNodeData(field_ids.velocity);
   Kokkos::deep_copy(velocity_h, (double)(0.0));
 
-  nimble_kokkos::HostVectorNodeView acceleration_h = model_data.GetHostVectorNodeData(field_ids.acceleration);
-  Kokkos::deep_copy(acceleration_h, (double)(0.0));
+  auto acceleration = model_data.GetVectorNodeData("acceleration");
```

Comment:
acceleration is handled differently from other (nodal) vars like velocity, I assume this is an interim edit not something intrinsically different with acceleration. The simplified/direct syntax is appreciated

Reply:
@rjones-sandia 
Yes I have not updated completely 'nimble_kokkos.cc' because the other fields appear in computation of the internal forces. The section about internal force needs to be moved in `nimble_kokkos::ModelData`. This is the next step.
The main file (that will stay) is `nimble_mpi.cc`